kitsune/wiki/jinja2/wiki/document.html

```diff
@@ -70,6 +70,10 @@
       </div>
     </header>
 
+    {% if show_cta_banner %}
+      {{ sumo_cta_banner(text=_('Unlock more from the Firefox suite: Sync, Monitor, Lockwise and <a href="https://www.mozilla.org/firefox/accounts/">more</a>'),
```

Comment:
Do we sanitize this localised string to ensure it doesn't link somewhere we don't want, or include any `onclick` attributes, for instance?